src/deluge/gui/ui/sound_editor.cpp

```diff
@@ -560,6 +560,17 @@ void SoundEditor::updateSourceBlinks(MenuItem* currentItem) {
 	}
 }
 
+void SoundEditor::setupShortcutsBlinkFromTable(MenuItem const* const currentItem,
+                                               MenuItem const* const items[kDisplayWidth][kDisplayHeight]) {
+	for (auto x = 0; x < 15; ++x) {
+		for (auto y = 0; y < kDisplayHeight; ++y) {
+			if (items[x][y] == currentItem) {
+				setupShortcutBlink(x, y, 0);
```

Comment:
Should it return after this? There's cases where multiple pads have the same menu type, like audio clip reverse

Reply:
... yes